tools/cli/admin.go

```diff
@@ -272,8 +272,7 @@ func newAdminMembershipCommands() []cli.Command {
 		{
 			Name:  "list_db",
```

Comment:
should combine these 2 list (list_gossip and list_db) into just one list command, and show the discrepancy between the 2 if any.

Reply:
the latter is supposed to have pagination right whereas the former is not paginatable? 

i am thinking though to remove `list_gossip` since it is already included in `DescribeCluster` admin API. Then maybe move `list_db` under `cluster` commands and rename it

The final result would be:

`tctl admin cluster describe` - includes the result of `list_gossip` (has been like that for a while)
`tctl admin cluster list-members` - has pagination and lists the members. Has few filters

wdyt?

Reply:
this would still be a separate PR though, to keep the PRs atomic

Reply:
I think including members in DescribeCluster is less useful. And it is not easy to find difference between members from gossip and members from db. It usually indicate some cluster level configuration issues if there is discrepancy. I'm not sure how useful it is to list (potentially different) members from 2 commands. If I'm admin, I would be much more interest to see the difference of them and list them in one command so I don't have to manually run 2 commands and try to diff them.

Reply:
sg, also per direct discussion. I'm going to do this change though i would prefer to do it separately. Current PR is just to switch to the API